cmd/read_bin/main.go

```diff
@@ -0,0 +1,485 @@
+	"rfc4547-DOCSIS-Event-Notification.mib",
+	"rfc4560-Ping,-Traceroute,-and-Lookup.mib",
+	"rfc4560-Ping,-Traceroute,-and-Lookup2.mib",
+	"rfc4560-Ping,-Traceroute,-and-Lookup3.mib",
+	"rfc4624-Multicast-Source-Discovery-Protocol-MSDP.mib",
+	"rfc4625-Fibre-Channel-Routing-Information.mib",
+	"rfc4626-Fibre-Channel-s-Fabric-Shortest-Path-First-FSPF-Protocol.mib",
+	"rfc4631-Link-Management-Protocol-LMP.mib",
+	"rfc4639-DOCSIS-Cable-Device.mib",
+	"rfc4668-RADIUS-Authentication-Client-MIB-for-IPv6.mib",
+	"rfc4669-RADIUS-Authentication-Server-MIB-for-IPv6.mib",
+	"rfc4670-RADIUS-Accounting-Client-MIB-for-IPv6.mib",
+	"rfc4671-RADIUS-Accounting-Server-MIB-for-IPv6.mib",
+	"rfc4672-RADIUS-Dynamic-Authorization-Client-MIB.mib",
+	"rfc4673-RADIUS-Dynamic-Authorization-Server-MIB.mib",
+	"rfc4682-IPCDN-MTA.mib",
+	"rfc4706-ADSL2.mib",
+	"rfc4706-ADSL22.mib",
+	"rfc4711-Real-time-Application-Quality-of-Service-Monitoring-RAQMON.mib",
+	"rfc4712-Transport-Mappings-for-RAQMON-PDU.mib",
+	"rfc4747-Virtual-Fabrics.mib",
+	"rfc4750-OSPF-Version-2.mib",
+	"rfc4750-OSPF-Version-22.mib",
+	"rfc4780-Session-Initiation-Protocol-SIP.mib",
+	"rfc4780-Session-Initiation-Protocol-SIP2.mib",
+	"rfc4780-Session-Initiation-Protocol-SIP3.mib",
+	"rfc4780-Session-Initiation-Protocol-SIP4.mib",
+	"rfc4789-SNMP-over-IEEE-802.mib",
+	"rfc4801-TCs-for-GMPLS-Management.mib",
+	"rfc4802-GMPLS-TE.mib",
+	"rfc4802-GMPLS-TE2.mib",
+	"rfc4803-GMPLS-LSR.mib",
+	"rfc4803-GMPLS-LSR2.mib",
+	"rfc4805-DS1,-J1,-E1,-DS2,-and-E2-Interface-Types.mib",
+	"rfc4807-IPsec-SPD-configuration.mib",
+	"rfc4836-IEEE-802.mib",
+	"rfc4836-IEEE-8022.mib",
+	"rfc4837-Ethernet-Passive-Optical-Networks-EPON.mib",
+	"rfc4878-OAM-Functions-on-Ethernet-Like-Interfaces.mib",
+	"rfc4898-TCP-Extended-Statistics.mib",
+	"rfc4935-Fibre-Channel-Fabric-Configuration-Server.mib",
+	"rfc4936-Fibre-Channel-Zone-Server.mib",
+	"rfc4936-Fibre-Channel-Zone-Server2.mib",
+	"rfc4939-iSNS-Internet-Storage-Name-Service.mib",
+	"rfc4983-Fibre-Channel-Registered-State-Change-Notification-RSCN.mib",
+	"rfc5017-URI-TC.mib",
+	"rfc5060-Protocol-Independent-Multicast-PIM.mib",
+	"rfc5066-EFMCu-Interfaces.mib",
+	"rfc5066-EFMCu-Interfaces2.mib",
+	"rfc5097-UDP-Lite-Protocol.mib",
+	"rfc5098-PacketCable__IPCablecom-NCS-Signaling.mib",
+	"rfc5131-Textual-Convention-for-Language-Tags-LANGTAG-TC-.mib",
+	"rfc5132-IP-Multicast-IP-MCAST.mib",
+	"rfc5190-Middlebox-Communication-MIDCOM.mib",
+	"rfc5240-Protocol-Independent-Multicast-PIM-Bootstrap-Router.mib",
+	"rfc5324-Fibre-Channel-Security-Protocols-FC-SP.mib",
+	"rfc5324-Fibre-Channel-Security-Protocols-FC-SP2.mib",
+	"rfc5324-Fibre-Channel-Security-Protocols-FC-SP3.mib",
+	"rfc5324-Fibre-Channel-Security-Protocols-FC-SP4.mib",
+	"rfc5324-Fibre-Channel-Security-Protocols-FC-SP5.mib",
+}
+
+func main() {
+
+	gosmi.Init()
+	defer gosmi.Exit()
+
+	for _, name := range moduleNames {
+		//fmt.Printf("load moduel from bin: %s\n", name)
+		data, err := Asset(name)
+		if err == nil {
```

Comment:
good practice to check if `err != nil` and return early, then this block could be intended left.